apps/asap-server/src/controllers/discover.ts

```diff
@@ -98,7 +99,12 @@ export default class Discover implements DiscoverController {
     return {
       aboutUs: content.flatData.aboutUs || '',
       training: content.flatData.training?.map(parseGraphQLNewsAndEvents) ?? [],
-      members: content.flatData.members?.map(parseGraphQLUser) ?? [],
+      members:
+        content.flatData.members?.map((member) =>
+          parseGraphQLUser(
+            member as NonNullable<FetchUserQuery['findUsersContent']>,
```

Comment:
I was hoping to avoid casts like this. Is there no way around this?

Reply:
i could add a @todo to finish this ticket https://trello.com/c/5vW6YMsL/1694-replace-graphql-types-with-the-generated-ones-news